spec/Section 5 -- Validation.md

```diff
@@ -1853,6 +1939,41 @@ fragment isHouseTrainedFragment on Dog {
 This document is not valid because {queryWithExtraVar} defines an extraneous
 variable.
 
+### All Fragment Arguments Used
+
+**Formal Specification**
+
+- For every {fragment} in the document:
+  - Let {arguments} be the arguments defined by that {fragment}.
+  - Each {argument} in {arguments} must be used at least once in the fragment's
+    scope.
+
+**Explanatory Text**
+
+All arguments defined by a fragment must be used in that same fragment. Because
+fragment arguments are scoped to the fragment they're defined on, if the
+fragment does not contain a variable with the same name as the argument, then
+the argument is superfluous.
+
+For example the following is invalid:
```

Comment:
add comma after 'For example'
https://linguaholic.com/linguablog/comma-after-for-example/